aws/available-versions

```diff
@@ -237,11 +238,1394 @@
 2.0.3
 2.0.2
 2.0.1
+2.0.0dev0
```

Comment:
can we improve our filter-login in the github action workflow to filter out such versions (alpha, dev, pre, rc, beta, unstable, test, etc.)